src/pages/home/home.html

```diff
@@ -42,14 +42,17 @@ <h1 text-center>{{ disciplina.titulo }}</h1>
         <h2 class="titulo">Média final</h2>
       </div>
 
-      <previsao-nota titulo="Necessário tirar na AP1" [nota]="disciplina.notas.notaParaAP1"></previsao-nota>
-      <previsao-nota titulo="Necessário tirar na AP2" [nota]="disciplina.notas.notaParaAP2"></previsao-nota>
-      <previsao-nota titulo="Necessário tirar na AP3" [nota]="disciplina.notas.notaParaAP3"></previsao-nota>
+      <previsao-nota *ngIf="disciplina.notas.notaParaAP1 > 0" titulo="Necessário tirar na AP1" [nota]="disciplina.notas.notaParaAP1"></previsao-nota>
+      <previsao-nota *ngIf="disciplina.notas.notaParaAP2 > 0" titulo="Necessário tirar na AP2" [nota]="disciplina.notas.notaParaAP2"></previsao-nota>
+      <previsao-nota *ngIf="disciplina.notas.notaParaAP3 > 0" titulo="Necessário tirar na AP3" [nota]="disciplina.notas.notaParaAP3"></previsao-nota>
     </ion-card-content>
 
-    <ion-card-content *ngIf="disciplina.notas.notaParaAP3 > 10" text-center>
+    <ion-card-content class="titulo" *ngIf="disciplina.notas.notaParaAP3 > 10 || disciplina.notas.notaParaAP2 > 10 || disciplina.notas.notaParaAP1 > 10" text-center>
       Não foi dessa vez, sorte na próxima....
     </ion-card-content>
+    <ion-card-content class="titulo" *ngIf="disciplina.notas.notaParaAP3 < 0 || disciplina.notas.notaParaAP2 < 0 || disciplina.notas.notaParaAP1 < 0" text-center>
+      Parabéns, já está passado.
```

Comment:
Acho que seria interessante a gente avisar ao cara que se ele for aprovado na AP1 e na AP2, mesmo estando aprovado, ele tem que fazer a AP3. Porque se ele não fizer a AP3 ele pode perder.